configure

```diff
@@ -255,6 +257,18 @@ for (const board of boards) {
     })
 }
 
+const ibomJs = globule.find(['src/interactive_bom/InteractiveHtmlBom/*.js'])
+
+ninja
+  .rule('concat')
+  .run('cat $in > $out')
+  .description('Concatenate files')
```

Comment:
Hmmm, I believe this *overwrites* $out with the contents of $in.

Reply:
I think that's what I want here: `$in` is a sequence of JS files that need to be concatenated into a single file. The problem this solves is that IBOM uses a few Javascript files that are set up without module exports, i.e. they're just a list of functions. That means you can't `require` them in a sensible way, as far as I know. So what I'm doing is just bundling all those files up together into a single one to be loaded in the HTML wrapper for the IBOM page. It's not very elegant, but it's the best idea I could come up with so far.

Reply:
Ok, but I am still confused. `cat a > b` is the equivelent to `cp a b`. `cat a >> b` would *append* contents of `a` to `b`, is that what you want?

Reply:
OK, I see the confusion: that `$in` thing is *not* a single filename, it's an *array* of filenames, so `cat $in > $out` concatenates all those files into a single file. It's not just a copy of a single file.